src/ddsrt/src/string.c

```diff
@@ -144,12 +144,12 @@ ddsrt_strlcat(
 }
 
 void *
-ddsrt_memdup(const void *src, size_t n)
+ddsrt_memdup(const void *ptr, size_t len)
```

Comment:
I think `src` is more descriptive than `ptr` and I think `n` is a better choice than `len`:
* `len` has some association with string lengths, though to be fair no-one would be confused with a function called `memdup`. The C standard uses `n` for functions like `memcpy`
* `ptr` is very generic, but the risk of confusion is near zero because of the function name and because of the `const`ness of it

Reply:
I agree that this names can confuse. Fixed.